mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -113,6 +113,7 @@ CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
 	FIELD field_28265 POSITION_SOURCE_TYPE Lnet/minecraft/class_2378;
 	FIELD field_28266 GAME_EVENT_KEY Lnet/minecraft/class_5321;
 	FIELD field_29076 FLOAT_PROVIDER_TYPE Lnet/minecraft/class_2378;
+	FIELD field_33093 INT_PROVIDER_TYPE Lnet/minecraft/class_2378;
```

Comment:
consider mappping its registry key as `INT_PROVIDER_TYPE_KEY`

Reply:
Should I include FloatProviders in this PR?

Reply:
hmm, guess just have the name fixes in a new pr

Reply:
Heh, turns out not mapping `INT_PROVIDER_TYPE_KEY` nukes javadoc because stupid stitch inserts this wrong inferred name, which conflicts with already mapped name and breaks javadoc generation service